### ISSUE TITLE
Allow disabling default preferred aliases with `tagNamePreference`

### ISSUE THREAD

**Issue by tjrgg** [gajus/eslint-plugin-jsdoc#500]
I'd like to be able to disable some of the default preferred aliases and not have eslint fix them.

For example, I'd like to use both `@class` and `@constructor` in my code, but I don't see a way of doing that with this plugin. The `tagNamePreference` setting seems to only allow one option.

If there is a way to do this that I may have missed, please let me know. Otherwise, I would love to see this as an option. Perhaps this could be done with the `tagNamePreference` setting by using `true` to indicate the tag is acceptable (opposite of `false`)?

**Comment by brettz9** [comment]
You can set an alias to itself. (Per solution at https://github.com/gajus/eslint-plugin-jsdoc/issues/488#issuecomment-580732246 )

```js
"settings": {
        "jsdoc": {
            "tagNamePreference": {
                "constructor": "constructor",
            }
        }
    }
```

(I'm added documentation for this approach in a branch (destructuring) that appears close to ready for merging. I'd also be open to a PR allowing for `true` as well, but I think the important thing is for it to be documented.)

Closing as should resolve, but feel free to comment further as needed.
